geomstats/stiefel.py

```diff
@@ -0,0 +1,279 @@
+"""
+Stiefel manifold St(p,n),
+a set of all orthonormal p-frames in n-dimensional space,
+where p <= n
+"""
+
+import logging
+import random
+
+import geomstats.backend as gs
+
+from geomstats.embedded_manifold import EmbeddedManifold
+from geomstats.riemannian_metric import RiemannianMetric
+
+TOLERANCE = 1e-6
+EPSILON = 1e-6
+
+def matrix_f(sq_mat, f):
```

Comment:
what is f? in matrix_f and as the variable f? avoid one-letter variable name as it is less readable, both in the function's name and as the variable name.